adapters/tlsio_openssl.c

```diff
@@ -592,8 +592,8 @@ static int openssl_static_locks_install(void)
 
     if (openssl_locks != NULL)
     {
-        LogError("Locks already initialized");
-        result = MU_FAILURE;
+        LogInfo("Locks already initialized");
+        result = 0;
```

Comment:
This is concerning. It is a significant change from the original behavior. An we just ignore these errors on valgrind when it runs the long haul tests?

Reply:
reverting